example/objective_c/README.md

```diff
@@ -0,0 +1,36 @@
+# Objective C example
+
+This example shows how to use ffigen to generate bindings for an Objective C
+library. It uses the AVFAudio framework to play audio files.
+
+```
+dart play_audio.dart test.mp3
+```
+
+## Config notes
+
+The ffigen config for an Objective C library looks very similar to a C library.
+The most important difference is that you must set `language: objc`. If you want
+to filter which interfaces are included you can use the `objc-interfaces:`
+option. This works similarly to the other filtering options.
+
+It is recommended that you filter out just about everything you're not
+interested in binding (see the ffigen config in [pubspec.yaml](./pubspec.yaml)).
+Virtually all Objective C libraries depend on Apple's internal libraries, which
+are huge. Filtering can reduce the generated bindings from millions of lines to

```

Comment:
Question:

Does objectiveC respect the `include-directives:`? e.g. can users also opt to allow-list some of Apple library header files but skip the rest? (It's probably still cleaner to allow-list the individual interfaces, but `include-directives:` should be an option.)

Reply:
Yeah, every header, C or ObjC, goes through `shouldIncludeRootCursor`, which checks all that stuff.